packages/shared/routes/login/Login.svelte

```diff
@@ -19,9 +20,19 @@
     let state: LoginState = LoginState.Init
     let stateHistory = []
 
-    const _next = (event) => {
+    onMount(() => {
+        if (get(activeProfileId)) {
```

Comment:
Just realized we have to check if the `activeProfileId` exists, people can delete profiles directly removing the folder in their system

Reply:
resolved